### PR TITLE
Volo's Guide races, Unearthed Arcana, and a few fixes

### PR DESCRIPTION
**Pull request by Wemes**
Added Sacred Oath Type to Oathbreaker
Fixed Typos
Adjusted Keys to monster races that have PC counterparts
Fixed issue with Character Advancement Ability Score bonuses
Made Half Elf racial traits versatile in accordance with SCAG alternate rules
Made Shortsword a monk weapon
Fixed Skulker feat to use Dex for requirement
Fixed Water Genasi swim speed bonuses
Added the rest of the Volo's Guide PC races
Implemented Unearthed Arcana Artificer class
Implemented Unearthed Arcana Barbarian Primal Paths

### CODE REVIEW

File: data/5e/wizards_of_the_coast/core/players_handbook/ph_abilities.lst [hunk 1]
```diff
@@ -246,7 +246,13 @@ Tinker's Tools			KEY:Tool Proficiency ~ Tinker's Tools		SORTKEY:1_Race	CATEGORY:
 
 ###Block: Half-Elf
 # Ability Name			Unique Key						SORTKEY		Category of Ability		Type											Description																																										Stackable?	Multiple?	Choose					Ability									Bonus Ability Pool				Stat bonus				Modify VAR						Source Page
-Racial Traits ~ Half-Elf											CATEGORY:Internal																																																																				DEFINE:HalfElfSkillVersatility|0	BONUS:VAR|HalfElfSkillVersatility|1|TYPE=Base|PREFACT:1,ABILITIES,HalfElfSkillVersatility=True	ABILITY:Half-Elf Trait|AUTOMATIC|Half-Elf ~ Ability Score Increase	ABILITY:Half-Elf Trait|AUTOMATIC|Half-Elf ~ Darkvision			ABILITY:Half-Elf Trait|AUTOMATIC|Half-Elf ~ Skill Versatility|PREVAREQ:HalfElfSkillVersatility,0		ABILITY:Half-Elf Trait|AUTOMATIC|Half-Elf ~ Languages			ABILITY:Half-Elf Trait|AUTOMATIC|Half-Elf ~ Fey Ancestry		
+Racial Traits ~ Half-Elf											CATEGORY:Internal
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/core/players_handbook/ph_abilities.lst]: Multi-line is for development, this should be on single line.

File: data/5e/wizards_of_the_coast/core/players_handbook/ph_equip.lst [hunk 2]
```diff
@@ -74,7 +74,7 @@ Morningstar								KEY:Morningstar				PROFICIENCY:WEAPON|Morningstar		TYPE:Weapo
 Pike									KEY:Pike					PROFICIENCY:WEAPON|Pike				TYPE:Weapon.Resizable.Melee.Manufactured.Weapon_Melee.Piercing.Martial																			COST:5	WT:18					CRITRANGE:1	DAMAGE:1d10					WIELD:TwoHanded			SIZE:M
 Rapier								KEY:Rapier					PROFICIENCY:WEAPON|Rapier			TYPE:Weapon.Resizable.Melee.Manufactured.Weapon_Finesse.Piercing.Martial																		COST:25	WT:2					CRITRANGE:1	DAMAGE:1d8					WIELD:OneHanded			SIZE:M
 Scimitar								KEY:Scimitar				PROFICIENCY:WEAPON|Scimitar			TYPE:Weapon.Resizable.Melee.Manufactured.Weapon_Finesse.Slashing.Martial																		COST:25	WT:3					CRITRANGE:1	DAMAGE:1d6					WIELD:OneHanded			SIZE:M
-Shortsword								KEY:Shortsword				PROFICIENCY:WEAPON|Shortsword			TYPE:Weapon.Resizable.Melee.Manufactured.Weapon_Finesse.Piercing.Martial																		COST:10	WT:2					CRITRANGE:1	DAMAGE:1d6					WIELD:OneHanded			SIZE:M
+Shortsword								KEY:Shortsword				PROFICIENCY:WEAPON|Shortsword			TYPE:Weapon.Resizable.Melee.Manufactured.Weapon_Finesse.Piercing.Monk.Martial																		COST:10	WT:2					CRITRANGE:1	DAMAGE:1d6					WIELD:OneHanded			SIZE:M
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/core/players_handbook/ph_equip.lst]: This causes a miscalculation error.

File: data/5e/wizards_of_the_coast/core/players_handbook/ph_profs_weapon.lst [hunk 3]
```diff
@@ -32,7 +32,7 @@ Morningstar			TYPE:Weapon.Martial
 Pike				TYPE:Weapon.Martial
 Rapier			TYPE:Weapon.Martial
 Scimitar			TYPE:Weapon.Martial
-Shortsword			TYPE:Weapon.Martial
+Shortsword			TYPE:Weapon.Martial.Monk
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/core/players_handbook/ph_profs_weapon.lst]: Shortsword gets a double bonus in certain circumstances, this was removed to follow the fix the pcgen team implemented.

File: data/5e/wizards_of_the_coast/supplement/unearthed_arcana/artificer/_artificer.pcc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+TEMPLATE:ua_art_templates.lst
+WEAPONPROF:ua_art_profs_weapon.lst
+
+
+###Block: Revisions
+# BD - 0.01 Initial update 
+
+INFOTEXT:Supports 6.4.1
+# ENTRY DATE: 2015-01 (Unknown)
+# LST MONKEY: Bahamut
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/supplement/unearthed_arcana/artificer/_artificer.pcc]: This information is incorrect. I am not the LST monkey.

File: data/5e/wizards_of_the_coast/supplement/elemental_evil_players_companion/eepc_abilities.lst [hunk 5]
```diff
@@ -44,7 +44,7 @@ Reach to the Blaze	KEY:Genasi ~ Reach to the Blaze	SOURCEPAGE:p.10	CATEGORY:Spec
 Ability Score Increase	KEY:Water Genasi ~ Ability Score Increase	SOURCEPAGE:p.10	CATEGORY:Special Ability	SORTKEY:1_Race	TYPE:Race Trait.Genasi Trait.Water Genasi Trait.SpecialQuality	DESC:Your Wisdom score increases by 1.		BONUS:STAT|WIS|1|TYPE=Racial
 Acid Resistance		KEY:Genasi ~ Acid Resistance		SOURCEPAGE:p.10	CATEGORY:Special Ability	SORTKEY:1_Race	TYPE:Race Trait.Genasi Trait.Water Genasi Trait.SpecialQuality	DESC:You have resistance to acid damage.
 Amphibious			KEY:Genasi ~ Amphibious			SOURCEPAGE:p.10	CATEGORY:Special Ability	SORTKEY:1_Race	TYPE:Race Trait.Genasi Trait.Water Genasi Trait.SpecialQuality	DESC:You can breathe air and water.
-Swim				KEY:Genasi ~ Swim				SOURCEPAGE:p.10	CATEGORY:Special Ability	SORTKEY:1_Race	TYPE:Race Trait.Genasi Trait.Water Genasi Trait.SpecialQuality	DESC:You have a swimming speed of 30 feet.	BONUS:VAR|Race_Swim|30|TYPE=Base
+Swim				KEY:Genasi ~ Swim				SOURCEPAGE:p.10	CATEGORY:Special Ability	SORTKEY:1_Race	TYPE:Race Trait.Genasi Trait.Water Genasi Trait.SpecialQuality	DESC:You have a swimming speed of 30 feet.	MOVE:Swim,30
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/supplement/elemental_evil_players_companion/eepc_abilities.lst]: Is this not working with the preformula system?

File: data/5e/wizards_of_the_coast/supplement/unearthed_arcana/barbarian_primal_paths/_barbarian.pcc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ABILITY:ua_bpp_abilities.lst
+ABILITYCATEGORY:ua_bpp_abilitycategories.lst
+
+
+###Block: Revisions
+# BD - 0.01 Initial update 
+
+INFOTEXT:Supports 6.4.1
+# ENTRY DATE: 2015-01 (Unknown)
+# LST MONKEY: Bahamut
```
Comment (BahamutDragon) [inline data/5e/wizards_of_the_coast/supplement/unearthed_arcana/barbarian_primal_paths/_barbarian.pcc]: Again, this credit should be yours, not mine.

**Comment by Wemes** [comment]
Thanks BD, I've made changes as you suggested and committed them. Do I close this PR and make a new one?

**Comment by BahamutDragon** [comment]
You addressed everything except my question regarding the change to MOVE:Swim,30. 

The PR updates when you make changes, and will continue to do so as long as you push to this branch.

**Comment by Wemes** [comment]
Well, I'm not sure why the Bonus to swim it wasn't doing what it's supposed to be doing. If you have any ideas we can make it work.

**Comment by BahamutDragon** [comment]
I will investigate the swim issue later. Easy enough to correct this later if need be.

**Pull request merged**